docs/introduction/best-practices.md

```diff
@@ -71,51 +71,48 @@ purpose of using A-Frame.
 [geometrymerger]: https://www.npmjs.com/package/aframe-geometry-merger-component
 
 
-Performance is critical in VR. A high framerate must be maintained in order for
-people to feel comfortable. Here are some ways to help improve performance of
-an A-Frame scene:
-
-- Use [recommended hardware specifications][hardware].
-- Use the **[stats component][stats]** to keep an eye on various metrics (FPS,
-  vertex and face count, geometry and material count, draw calls, number of
-  entities). We want to maximize FPS and minimize everything else.
+Performance is critical in VR. People feel comfortable when you maintain a high frame rate.
+
+Check out the [recommended hardware specifications][hardware]. Generally, the fewer number of entities and lights in the scene, the better.
+
+Note that the **[stats component][stats]** provides various metrics (FPS, vertex and face count, geometry and material count, draw calls, number of entities). 
```

Comment:
Use is a better call to action than note I think

Reply:
```suggestion
Use the **[stats component][stats]** for various metrics (FPS, vertex and face count, geometry and material count, draw calls, number of entities). 
```